orchestrator/src/main/java/com/yahoo/vespa/orchestrator/status/ZookeeperStatusService.java

```diff
@@ -108,55 +96,16 @@ public MutableStatusRegistry lockApplicationInstance_forCurrentThreadOnly(Applic
     MutableStatusRegistry lockApplicationInstance_forCurrentThreadOnly(
             ApplicationInstanceReference applicationInstanceReference,
             long timeoutSeconds) {
-        Thread currentThread = Thread.currentThread();
-
-        //Due to limitations in SessionFailRetryLoop.
-        assertThreadDoesNotHoldLock(currentThread,"Can't lock " + applicationInstanceReference);
+        String lock2Path = applicationInstanceLock2Path(applicationInstanceReference);
```

Comment:
Why is the lock still having the number "2" in its name? I thought the number's purpose was to distinguish from the old lock.

Reply:
Fixed